### PR TITLE
Pull Request 

### PR DESCRIPTION
**Pull request by abouhid**
This pull request is necessary for finishing the Bubble-Sort Project using Ruby.
This project consists of creating a code that has 2 functions:
- One that orders an array numerically;
- One that orders an array by the length of the strings it contains.

Ruby knowledge and linters (rubocop) were used in this project.

### CODE REVIEW

**Comment by lekegitrepo** [comment]
#### Hi @abouhid and @Rocio01 :clap: :clap:
#### Good job :+1: However, the methods returned nil when I tried to test methods with RSpec. I suggest you should remove the print from the values implicitly returned by the method because this can cause a method to return nil.

**_fix this and submit for review._**

#### _Reviewed by Temitayo Adeleke._

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
#### Status: Request changes

**Comment by abouhid** [comment]
> #### Hi @abouhid and @Rocio01  
> #### Good job  However, the methods returned nil when I tried to test methods with RSpec. I suggest you should remove the print from the values implicitly returned by the method because this can cause a method to return nil.
> **_fix this and submit for review._**
> 
> #### _Reviewed by Temitayo Adeleke._

Hello Mr. Adeleke! Thank you for your quick response and attention.
We made a change on the print/puts inside the functions as you mentioned. However, we are not quite sure if it is exactly this that you meant since we did not know how to check if there is a 'nil' appearing. 
I hope everything is fine now. Thanks again!

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      end
+    end
+    array
+  else
+    puts 'not a valid array'
+  end
+end
+
+puts 'Result of bubble_sort():'
+printbubble_sort([4, -3, 37.3, -78, 2.4, 0, 4])
```
Comment (christianotieno) [inline bubble_sort.rb]: ## Clean Code
```suggestion
print bubble_sort([4, -3, 37.3, -78, 2.4, 0, 4])
```
Kindly add a space in between the print statement and the sort method.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project guys. :fireworks: 
##
***Most of the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***

- Before we merge these changes kindly take note on the minor suggested changes below.
Once you are done, you can now merge these changes. You don't have to submit for another code review.
- I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**